template/en/default/account/auth/verify_account_creation.html.tmpl

```diff
@@ -7,21 +7,57 @@
   #%]
 
 [% PROCESS global/header.html.tmpl
-   title = "Verify Account Creation" %]
+   title = "Verify Account Creation"
+   style_urls = [ 'extensions/BMO/web/styles/create_account.css' ]
+%]
 
 <h1>Verify Account Creation</h1>
 
-<p>An account for <em>[% login FILTER html %]
-  [% IF realname %]([% realname FILTER html %])[% END %]</em> was not
-  found already in the system. Please verify that you want a new account
-  to be created.</p>
+<div id="verify-auto-account-creation">
+  <p>An account for <em>[% login FILTER html %]
+    [% IF realname %]([% realname FILTER html %])[% END %]</em> was not
+    found already in the system. Please verify that you want a new account
+    to be created.</p>
 
-<div class="verify-auto-account-creation">
   <form method="POST" action="[% basepath FILTER none %]token.cgi">
     <input type="hidden" name="t" value="[% token FILTER html %]">
     <input type="hidden" name="a" value="verify_auto_account_creation">
     <input type="submit" name="verify" value="Verify">
   </form>
 </div>
 
+<div id="right-blurb">
+  <strong>
+    There are a few things to know about using this site.
+  </strong>
+  <ul>
+    <li>
+      Please read our
+      <a href="[% basepath FILTER none %]page.cgi?id=bug-writing.html" target="_blank" rel="noopener noreferrer">[% terms.Bug %] Writing Guidelines</a>.
+    </li>
+    <li>
+      [% terms.Bugzilla %] is a public place. Your comments and other activities
+      on [% terms.bugs %] will generally be publicly visible, and your email address will be
+      accessible through public APIs and will be visible to all logged-in users
+      of [% terms.Bugzilla %]. Some people use an
+      <a href="https://www.lifewire.com/top-free-email-services-1171481"
+      target="_blank">alternative email address</a> for this reason.
+      See Mozilla's <a href="https://www.mozilla.org/privacy/websites/"
+      target="_blank">Websites, Communications & Cookies Privacy Notice</a> for
+      more information on our privacy policies.
+    </li>
+    <li>
+      When using [% terms.Bugzilla %] to submit patches, comments, code, and any other content,
+      you agree to our policies on open source licensing and content submission. See Mozilla’s
+      <a href="https://www.mozilla.org/about/legal/terms/mozilla/" target="_blank">Websites
+      and Communications Terms of Use</a> for more information.
+    </li>
+    <li>
+      You understand that your conduct on this site is subject to both <a href="[% basepath FILTER none %]page.cgi?id=etiquette.html">[% terms.Bugzilla %] Etiquette</a>,
+      and the <a href="https://www.mozilla.org/about/governance/policies/participation/">Mozilla Community Participation Guidelines</a>.
+      By creating an account, you agree to abide by them.
+    </li>
+  </ul>
+</div>
+
```

Comment:
As this is an exact copy from the content in `extensions/BMO/template/en/default/account/create.html.tmpl` can we shift it into its own file and include it into both locations?

I'm concerned that in the future we'll update one location and not the other.